solution/ui/regulations/css/scss/_application_settings.scss

```diff
@@ -263,6 +265,12 @@ $width-3xl: 1920px; // The largest the site container will get
     color: $secondary_indicator_text;
 }
 
+.tertiary-indicator {
+    background: $tertiary-indicator_background;
+    color: $mid-gray;
```

Comment:
it's interesting that you've used `$mid-gray` with a dash as the variable name, because the variable is defined on line 21 with an underscore: `$mid_gray`.  

And yet, even though you've used a dash, it is working and compiling to the correct color.

I was confused as to how this could happen, but after a little research I've learned something new: in Sass, it doesn’t matter whether your variable has an underscore or a dash - [they work both ways](https://pyronaur.com/dashes-and-underscores-in-sass-scss/).  Today I learned!

Still, it might be clearer if these are changed to `$mid_gray` with an underscore to match the actual variable declaration.